main.cpp

```diff
@@ -16,31 +16,67 @@ video: Chapter 2 - Part 3
  
  1) Write down the names of all of the primitives available in C++ (excluding wchar_t)
  put them here: 
- 
- 
- 
- 
- 
- 
- 
- 
- 
- 
+    Integer int
+    Character char
+    Boolean bool
+    Floating Point float
+    Double Floating Point double
+    Valueless or Void Voic
+    Wide Character whar_t
+
  
 2) for each primitive type, write out 3 variable declarations inside the variableDeclaration function.
     give each declaration an initial value
         just ignore wchar_t. you do not need to declare 3 variables of type 'wchar_t'
         'void' is a return type. you do not need to declare 3 variables of type 'void'.
     at the end of the function, pass each variable to the ignoreUnused function
  
+        int x = 1;
+        int myInt = 5;
+        int B = 10;
+        char myChar = 'd';
+        char myGrade = 'A';
+        char str = 'a';
+        bool isMusicFun = true;
+        bool isHotToday = true;
+        bool amIYoung = false;
+        double myDouble = 7.88;
+        double newDouble = 18.66;
+        double myDoubleNum = 1.88;
+        float myFloat = 0.1f
+        float hisFloat = 9.6f
+        float theirFloat = 55.55f
+        ignoreunused(x, myInt, B, myChar, myGrade, str, isMusicFun, isHotToday, amIYoung, myDouble, newDouble, myDoubleNum, myFloat, hisFloat, theirFloat)
```

Comment:
These go in the function named `variableDeclarations` down below.
- [ ] move them.